### DISCUSSION TITLE
Change background grid in graphs according to color scheme

### DISCUSSION THREAD

**Discussion by ckafi** [monkeytypegame/monkeytype#4804] [Ideas] [answered]
Hi,

right now the color of the background grid of the graphs in the account page is always a muted dark grey. This makes it hard to impossible to see it when using a darker color scheme. It would be nice if the grid changed color, the it's visible with all color schemes.

Thanks.

**Comment by Miodec** [comment, marked as answer]
Sure, ill update it.

**Reply by ckafi** [reply]
Not to nag, but any ETA on this?

**Reply by Miodec** [reply]
already implemented
